frontend/src/components/Modal/Modal.scss

```diff
@@ -27,7 +27,8 @@
   bottom: 0;
   left: 0;
   right: 0;
-  width: 100%;
+  max-height: 100vh;
```

Comment:
should be minus the header height, see `calc` function of css
also should only be like 70vh on mobile might look nicer

Reply:
I tried the 70vh and the lowest I found that I could adjust it with the minus header-height was 85% or vh. If I made it less it cuts off the social media icons.